src/olympia/devhub/views.py

```diff
@@ -1185,13 +1185,13 @@ def version_delete(request, addon_id, addon):
         messages.success(
             request,
             ugettext('Version %s disabled.') % version.version)
-        version.is_user_disabled = True
+        version.is_user_disabled = True  # Will update the files/activity log.
```

Comment:
I know this isn't new code, but shouldn't there be a `version.save` somewhere to store the updated value? It's a little worrying.

Reply:
That's why I added the comment, but obviously it wasn't clear enough :) I had the same concern, and it turns out that `is_user_disabled` is a `@property` with a `setter` that directly calls `.update()` on the attached files...

Reply:
Ah.  (Aaaaand, oh look - I wrote `version.is_user_disabled` with it's setter. D'oh)